src/ActionModal/components/DelegationModal.vue

```diff
@@ -312,7 +322,13 @@ export default {
       amount: {
         required: x => !!x && x !== `0`,
         decimal,
-        between: between(SMALLEST, this.maxAmount)
+        max: x => Number(x) <= this.maxAmount,
+        min: x => Number(x) >= SMALLEST,
+        maxDecimals: x => {
+          return x.toString().split(".").length > 1
```

Comment:
just making sure: this serializes always to american number format?

Reply:
oops... let me check that

Reply:
Yeah, it is good. It is impossible to input it with commas 👍